serverless/messaging/api-cli/python-node-sns.mdx

```diff
@@ -112,7 +112,7 @@ _, _ = awsSns.Subscribe(&sns.SubscribeInput{
   TopicArn: createTopicResponse.TopicArn,
 })
 
-You can also set up an SQS queue from the same namespace, as a [Dead Letter Queue](/serverless/messaging/concepts/#dead-letter-queue):
+You can also set up an SQS queue from the same Scaleway Project, as a [Dead Letter Queue](/serverless/messaging/concepts/#dead-letter-queue):
```

Comment:
Is this correct?

Reply:
No :-/

Idem for Dead Letter Queues. Not supported anymore.